src/tfr_communication/package.xml

```diff
@@ -15,6 +15,7 @@
   <test_depend>gtest</test_depend>
   <depend>roscpp</depend>
   <depend>std_msgs</depend>
+  <depend>tfr_msgs</depend>
```

Comment:
I think for most cases @collinsc I agree with you. I think this is the one exception because it's one of our unified packages that's meant to be shared throughout the whole project, and I think we'll be using it so much that leaving it out will just result in longer debug times trying to find out where the build is breaking when it'll just inevitably be added in again for almost every package. For cases aside from this one though, I agree, we should keep that down to a minimum.